### PR TITLE
Add pathview as a standalone tool

### PR DESCRIPTION
**Pull request by bebatut**
FOR CONTRIBUTOR:
* [X] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] License permits unrestricted use (educational + commercial)
* [X] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/pathview/pathview.xml [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+ls .
+    ]]></command>
+    <inputs>
+        <conditional name="pathway">
+            <param name="nb" type="select" label="Number of pathways to plot">
+                <option value="one" selected="true">One</option>
+                <option value="multiple">Multiple</option>
+            </param>
+            <when value="one">
+                <param name="one_id" type="text" label="KEGG pathway"/>
```
Comment (bgruening) [inline tools/pathview/pathview.xml]: add a validator?

File: tools/pathview/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+owner: iuc
+name: pathview
+description: Pathview is a tool set for pathway based data integration and visualization. 
+long_description: |
+   t maps and renders a wide variety of biological data on relevant pathway graphs. All users need is to supply their data and specify the target pathway. Pathview automatically downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, Pathview also seamlessly integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.
```
Comment (bgruening) [inline tools/pathview/.shed.yml]: something is wrong here ...

File: tools/pathview/pathview.xml (hunk 1 above)
Comment (bebatut) [reply tools/pathview/pathview.xml]: I added one

File: tools/pathview/.shed.yml (hunk 2 above)
Comment (bebatut) [reply tools/pathview/.shed.yml]: I should be better now

**Comment by bgruening** [comment]
Merci!

**Pull request merged**